x/leverage/keeper/oracle.go

```diff
@@ -11,6 +11,9 @@ import (
 
 var ten = sdk.MustNewDecFromStr("10")
 
+// TODO: Parameterize and move this
+const numHistoracleStamps = uint64(10)
```

Comment:
Rather than adding an extra parameter for this would it work to just use `MaximumMedianStamps` since that param's value should be set in a way where we're only storing the amount of median data leverage needs?

Reply:
Maybe - but if we decide in the future to have ibc rate limiting, or some other feature, use a longer amount than leverage, if would be good to have this param at the ready.